packages/vscode-ui5-language-assistant-bas-ext/scripts/copy.js

```diff
@@ -0,0 +1,30 @@
+const { copySync, readdirSync } = require("fs-extra");
+const { join } = require("path");
+
+const getBaseSrc = () =>
+  join(__dirname, "..", "..", "vscode-ui5-language-assistant");
+const getBaseDes = () => join(__dirname, "..");
+
+const srcReadme = join(getBaseSrc(), "README.md");
```

Comment:
This is a technical npmjs package.
So is the readme designed for vscode marketplace really suitable? should it have a small README of its own instead?